src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs

```diff
@@ -804,5 +804,10 @@ private void UpdateCheckboxVisibility(object sender, bool isPointerOver)
 					VisualStateManager.GoToState(userControl, "HideCheckbox", true);
 			}
 		}
+
+		private void TextBlock_IsTextTrimmedChanged(TextBlock sender, IsTextTrimmedChangedEventArgs e)
+		{
+			ToolTipService.SetToolTip(sender, sender.IsTextTrimmed ? sender.Text : null);
```

Comment:
I assume, we only do this because of UX, not because of performance reasons?

Reply:
That's right, we only want the tooltip to show if the text is trimmed.